src/db/scripts/farm-migration.ts

```diff
@@ -0,0 +1,110 @@
+import { db } from "../db";
+import {
+  Devices,
+  FarmDatabaseInsertType,
+  RewardSplits,
+  RewardSplitsInsertType,
+  farms,
+} from "../schema";
+
+export interface MigrationFarmData {
+  auditor: string;
+  auditor_percentage_glow_weight: number;
+  auditor_percentage_usdc_weight: number;
+  comments: string[] | null;
+  glow_weight: number;
+  payment_tx_hash: string;
+  payout_wallet: string;
+  protocol_fee: number;
+  protocol_fee_payment_hash: string;
+  short_id: number;
+  timestamp_audited_completed: number;
+  hexlified_public_key: string;
+  installer_glow_fee_percent: number;
+  installer_usdg_fee_percent: number;
+  installer_wallet: string;
+  old_short_ids: string[];
+}
+
+export const insertFarmWithDependencies = async (
+  farmData: MigrationFarmData
+) => {
+  return db.transaction(async (tx) => {
+    const farmId = await tx
+      .insert(farms)
+      .values({
+        gcaId: farmData.auditor,
```

Comment:
this needs to be Jared. Auditor in the json refers to the glow solutions people.
This is Jared's wallet:
0xB2d687b199ee40e6113CD490455cC81eC325C496